libraries/standard/mqtt/src/private/iot_mqtt_helper.h

```diff
@@ -71,7 +71,17 @@
  * @param[in] x The unsigned int to check.
  * @param[in] position Which bit to check.
  */
-#define UINT8_CHECK_BIT( x, position )    ( ( ( x ) & ( 0x01U << ( position ) ) ) == ( 0x01U << ( position ) ) )
+#define UINT8_CHECK_BIT( x, position )    ( ( ( x ) &( 0x01U << ( position ) ) ) == ( 0x01U << ( position ) ) )
```

Comment:
I think this formatting is slightly off.

Reply:
I'll run uncrustify again.